### ISSUE TITLE
Add tooltip in bottom navigations

### ISSUE THREAD

**Issue by harshmangalam** [harshmangalam/qwik-x#26]
add tooltip in icon button of bottom navigations

**Comment by RABBI31** [comment]
Hello,
I read your issue and find out that i can solve it if you assign me.
Thank you

**Comment by MokshagnaV** [comment]
Hey guys,
I have worked on bottom navigation before. I thought of adding tool tip at first (As we know, Tool tip will be visible when user hovers on it). But, this bottom navigation is for mobile device(Small screens) right. In mobile screen hover is not possible, Because it is mostly a touch screen for small devices.
So, i have added small text below the icons instead of tool tip in #1.
I don't know how, it's not right now.

**I suggest you guys to add small text at bottom of icons or any other design install of tooltip**

**Comment by harshmangalam** [comment]
@MokshagnaV I have added small modifications in bottom navigations and  instead of adding text bottom to icon tooltip will provide better UX.

Utilize daisyui Tooltip .

**Comment by MokshagnaV** [comment]
Yeah I added tooltip first. But in touch screen, we cannot hover right. If we add tooltip, the tooltip text will appear when user clicks on it(on touch screens).

**Comment by harshmangalam** [comment]
Yes mobile screen user will not see tooltip but small device media can see if they have pointer instead of touch
